components/html-rendering/src/test/resources/benchmark-renderings/HtmlAnalysisResultWriterTest-testEmptyResult.html

```diff
@@ -1,13 +1,35 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title>Analysis result</title>  <script type="text/javascript" src="http://eobjects.org/resources/datacleaner-html-rendering/analysis-result.js"></script>
-<link rel="shortcut icon" href="http://eobjects.org/resources/datacleaner-html-rendering/analysis-result-icon.png" />
+  <title>Analysis result</title>  <link rel="shortcut icon" href="http://eobjects.org/resources/datacleaner-html-rendering/analysis-result-icon.png" />
+<link rel="stylesheet" type="text/css" href="https://cdnjs.cloudflare.com/ajax/libs/jqueryui/1.12.0/themes/base/jquery-ui.css" />
+<link rel="stylesheet" type="text/css" href="http://eobjects.org/resources/datacleaner-html-rendering/analysis-result.css?load=script" />
 <script type="text/javascript">//<![CDATA[
-  var analysisResult = {};
+var analysisResult = {};
+requirejs = {
```

Comment:
I didn't worry too much about it, since we'll hopefully rework this as part of the monitor rewrite, but there's a good chance we'll skip this when we do that, so let's do it now.